compiler/luci/pass/src/CircleQuantizer.cpp

```diff
@@ -49,6 +50,148 @@ namespace
 using namespace luci;
 using LayerParam = luci::CircleQuantizer::Options::LayerParam;
 
+// This function updates user-given input_type to match with the input signature of graph
+// If user gives only one input_type, it will be expanded to the number of graph inputs
+void canonicalize_input_type(loco::Graph *g, std::vector<loco::DataType> &input_type)
+{
+  assert(g); // FIX_CALLER_UNLESS
```

Comment:
Sometimes our static analysis tool complains about this assert on release build. Maybe we can do nothing for this case.

```suggestion
  if (g == nullptr)
    return;
    
  // ..
  
  if (inputs == nullptr)
    return;
```